src/components/Card/Media.js

```diff
@@ -73,6 +73,7 @@ function Media({
     <div className={classes.root}>
       <Image
         layout="fill"
+        optimized="true"
```

Comment:
optimized=true is usually what causes pixelation @nerdlet. Are you sure that's what [this](https://www.pivotaltracker.com/story/show/181356820) story need?

Reply:
@kilemensi Um, I don't think we need it for now. Although I think we need better images because the current ones on metrics are blurry compared to the previous ones. https://github.com/CodeForAfrica/PesaYetu/blob/main/src/assets/Metric2%402x.png